packages/slice-machine/pages/_app.tsx

```diff
@@ -15,9 +15,11 @@ import Tracker from "@src/tracker";
 import "react-tabs/style/react-tabs.css";
 import "rc-drawer/assets/index.css";
 import "react-datepicker/dist/react-datepicker.css";
+import "react-toastify/dist/ReactToastify.css";
```

Comment:
already imported in packages/slice-machine/components/ToasterContainer/index.tsx

Reply:
oupsi something forgotten during the rebase :/

Reply:
fixed